bot/cogs/help_channels.py

```diff
@@ -546,19 +548,20 @@ async def move_to_dormant(self, channel: discord.TextChannel, caller: str) -> No
 
         self.bot.stats.incr(f"help.dormant_calls.{caller}")
 
-        if channel.id in self.claim_times:
-            claimed = self.claim_times[channel.id]
-            in_use_time = datetime.now() - claimed
+        claimed_timestamp = await self.claim_times.get(channel.id)
+        if claimed_timestamp:
+            claimed = datetime.fromtimestamp(claimed_timestamp)
+            in_use_time = datetime.utcnow() - claimed
             self.bot.stats.timing("help.in_use_time", in_use_time)
 
-        if channel.id in self.unanswered:
-            if self.unanswered[channel.id]:
+        unanswered = await self.unanswered.get(channel.id)
+        if unanswered is not None:
+            if unanswered:
                 self.bot.stats.incr("help.sessions.unanswered")
             else:
                 self.bot.stats.incr("help.sessions.answered")
```

Comment:
Thoughts of changing this the following?

```py
if unaswered:
    ...
elif unanswered is not None:
    ...
```

Reply:
Don't see why not.